bytestring.cabal

```diff
@@ -135,6 +135,22 @@ test-suite prop-compiled
                     -threaded -rtsopts
   default-language: Haskell2010
 
+test-suite strictness-checks
+  type:             exitcode-stdio-1.0
+  main-is:          Main.hs
+  other-modules:    Orphans Partial Prelude'
+  hs-source-dirs:   tests/strictness-checks
+  build-depends:    base, bytestring, deepseq,
+                    tasty, tasty-quickcheck, QuickCheck,
+                    transformers, Stream, pretty-show
```

Comment:
1. Do you need `Stream`? I cannot find `Data.Stream` imported. The package has not been updated for 6 years, which I deem a dangerous dependency.
2. The last `pretty-show`, which does not depend on `text` (which depends on `bytestring`), is 1.6.16, three years ago. Not too bad, but I wonder how important this dependency is.

Reply:
1. Oh, seems like I need to enable the warning for unused packages separately… I brought `Stream` in to carry around polymorphic labels for my errors, but then I made labels simply numbers and `Stream` is not needed anymore.

2. Not so important. Shrinking makes sure examples are small and easy to inspect. Imma drop it.

Reply:
2. How about I use [`groom`] instead? It does not depend on `text` or `bytestring` at all.

[`groom`]: https://hackage.haskell.org/package/groom

Reply:
Ping me if you hate `groom` to.

Reply:
`groom` depends on `haskell-src-exts`, which is pretty much no go. OK, let's keep `pretty-show` for a while, we can remove it later, if it starts causing build issues.